speed-dreams/source-2.2.3/src/modules/track/trackv1/track.cpp

```diff
@@ -135,6 +135,8 @@ GetTrackHeader(void *TrackHandle) {
     // Read header
     theTrack->name = GfParmGetStr(TrackHandle, TRK_SECT_HDR, TRK_ATT_NAME, "no name");
     theTrack->descr = GfParmGetStr(TrackHandle, TRK_SECT_HDR, TRK_ATT_DESCR, "no description");
+    /// SIMULATED DRIVING ASSISTANCE: speed limit can be read from the header
+    theTrack->SpeedLimit = GfParmGetStr(TrackHandle, TRK_SECT_HDR, TRK_ATT_AVERAGE_SPEEDLIMIT, "average speed limit");
```

Comment:
This is a number, so why not use `GfParmGetNum`?

Reply:
Should indeed be updated to GfParmGetNum, string makes no sense in this case.